### PR TITLE
WDB-12/User Profile

### PR DESCRIPTION
**Pull request by AArlauskas**
Tasks implemented:
1. Setup user profile page
2. Implemented "Save" button functionality
3. Added snackbars to inform users if the changes were saved successfully or not.

image

### CODE REVIEW

File: frontend/src/components/ChangeProfileForm/ChangeProfileForm.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            <Grid item>
+              <Avatar style={{ backgroundColor: SUNFLOWER }}>
+                {firstName[0] + lastName[0]}
+              </Avatar>
+            </Grid>
+            <Grid item>
+              <Typography variant="h6">{`${firstName} ${lastName}`}</Typography>
+            </Grid>
+          </Grid>
+        </Grid>
```
Comment (tmaciulis22) [inline frontend/src/components/ChangeProfileForm/ChangeProfileForm.js:113]: Add more spacing between this Grid item and form below. Now the top elements look squashed with email text field

File: frontend/src/containers/LibraryPage/LibraryPage.js [hunk 2]
```diff
@@ -14,8 +14,8 @@ class LibraryPage extends React.Component {
               /* [TM:] TODO WDB-29 */
             }}
             showAvatarAndLogout
-            firstName="Juris"
```
Comment (tmaciulis22) [inline frontend/src/containers/LibraryPage/LibraryPage.js:17]: RIP Juris

**Review by tmaciulis22 — APPROVED** [review]
Minor spacing change, but other than that everything is good 🚀

File: frontend/src/containers/LibraryPage/LibraryPage.js (hunk 2 above)
Comment (AArlauskas) [reply frontend/src/containers/LibraryPage/LibraryPage.js:17]: Goodbye, my Love,
Goodbye, my friend

File: frontend/src/components/ChangeProfileForm/ChangeProfileForm.js (hunk 1 above)
Comment (AArlauskas) [reply frontend/src/components/ChangeProfileForm/ChangeProfileForm.js:113]: done

**Review by povilassta — APPROVED** [review]

**Pull request merged**